src/main/java/com/duke/bot/task/Task.java

```diff
@@ -0,0 +1,37 @@
+package com.duke.bot.task;
+
+import java.util.Objects;
+
+public abstract class Task {
+    private String title;
+    private boolean isDone;
+
+    public Task(String title, boolean isDone) {
+        this.title = title;
+        this.isDone = isDone;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public abstract Task setDone(boolean isDone);
```

Comment:
Consider renaming `setDone` to something like `setDoneStatus` or `setDoneStatusAs`; `setDone` implies that regardless of the argument passed in, the task will always be set as done. 

Or since all calls to this method are `setDone(true)` , just change the method to take in no arguments.